docs/deployment/enable_sovereign_deployment.md

```diff
@@ -1,8 +1,8 @@
 # Enable Sovereign Region Deployment
 
-Follow these steps to enable a split deployment.  All resources outside of AOAI will be deployed in an Sovereign region.
+Follow these steps to enable a Sovereign region deployment.  If you need access to AOAI in a UsGov region please fill out this form https://aka.ms/AOAIgovaccess.
 
-Only Sovereign regions supported today are **US Gov**.
+Only Sovereign regions supported today are **US Gov Virginia, US Gov Arizona**.
```

Comment:
Rather than hard code this in markdown, can we link to a MS docs page that gives the list so that once it updates we don't have to keep updating our docs?

Reply:
Not yet, it would be this page but its not yet updated. https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?regions=usgov-non-regional,us-dod-central,us-dod-east,usgov-arizona,usgov-texas,usgov-virginia&rar=true&products=cognitive-services

Reply:
It is available here https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#azure-government-regions

Reply:
Should be updated now